### ISSUE TITLE
ERROR unable to set 'upload_max_size'

### ISSUE THREAD

**Issue by pchampio** [LycheeOrg/Lychee-Laravel-Docker#33]
I followed the https://github.com/LycheeOrg/Lychee-Laravel-Docker#advanced-configuration, and set a higher `upload_max_size`.
I'm still not able to upload >100M file, here is the error from `/var/log/nginx/error.log`
```
2020/02/24 20:09:59 [error] 75#75: *4 FastCGI sent in stderr: "Passing INI directive through FastCGI: unable to set 'upload_max_size'" while reading response header from upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/run/php/php7.3-fpm.sock:", host: "localhost"
```

**Comment by manniL** [comment]
Same issue while having the default config.

**Comment by manniL** [comment]
Update: resolved the issue through #41 

@pchampio To increase the size, increase the other params presented in https://github.com/LycheeOrg/Lychee-Laravel-Docker/blob/master/default.conf and	the `client_max_body_size 100M;`
